src/components/carousels/homepage-carousel/HomepageCarousel.jsx

```diff
@@ -0,0 +1,62 @@
+import './HomepageCarousel.css';
+import { useEffect, useState } from 'react';
+
+function HomepageHeaderCarousel({ slides }) {
+  const [current, setCurrent] = useState(0);
+  const [autoPlay, setAutoPlay] = useState(true);
+  let timeOut = null;
+
+  useEffect(
+    () => {
+      let timeOut =
+        autoPlay &&
+        setTimeout(() => {
+          slideRight();
+        }, 5000);
+      return () => clearTimeout(timeOut);
+    }
+    //timer function- disables error as timer doesn't need to be stored
+  );
+
+  const slideRight = () => {
+    setCurrent(current === slides.length - 1 ? 0 : current + 1);
+  }; // slide right and current checks what image it is on - it will go back to one if it reaches the end
+
+  return (
+    <div className='carousel-container'>
+      <div className='hph-carousel-background'>
+        <div
+          className='hph-carousel'
+          onMouseEnter={() => {
+            setAutoPlay(false); //checks
+            clearTimeout(timeOut);
+          }}
+          onMouseLeave={() => {
+            setAutoPlay(true);
+          }}
+        >
+          <div className='hph-carousel_wrapper'>
+            {slides.map((image, index) => {
+              return (
+                <div key={index} className={index === current ? 'hph-carousel_card hph-carousel_card-active' : 'hph-carousel_card'}>
```

Comment:
Looks great! There are a couple of classes with slightly different naming conventions (using underscores instead of dashes). You might want to have a look at them for the sake of consistency but it's something that can be altered at any point

Reply:
Will have a look now, thanks for the heads up

Reply:
Cool, that was a file that was supposed to be removed so thanks for letting us know